nengo/neurons.py

```diff
@@ -545,24 +545,20 @@ def __init__(
         tau_rc=0.02,
         tau_ref=0.002,
         amplitude=1,
-        **kwargs
     ):
         super(AdaptiveLIFRate, self).__init__(
-            tau_rc=tau_rc,
-            tau_ref=tau_ref,
-            amplitude=amplitude,
-            **kwargs)
```

Comment:
The `LIF` constructor was getting called, and `min_voltage` was getting properly set. I tested that, and so did `test_neurons.test_argreprs`. I'm not quite sure how it was happening, but this `**kwargs` was needed for it.

Anyway, the new way seems cleaner, so I'm fine with it.

Reply:
You're right, `super` looks through the MRO and call both `__init__` methods, so `AdaptiveLIF` was working correctly because `AdaptiveLIFRate` accepted and ignored `min_voltage` thanks to the `**kwargs` (this [SO thread](https://stackoverflow.com/questions/3277367/how-does-pythons-super-work-with-multiple-inheritance) helped clarify). I had removed the `**kwargs` before making the other changes so didn't realize I had broken it.